front-end/Dockerfile

```diff
@@ -1,12 +1,16 @@
 FROM node:lts-alpine
 
+ARG BUILD_ENV=prod
+
 RUN npm install -g http-server
 
 WORKDIR /usr/src/app
 COPY ./ ./
+COPY ./${BUILD_ENV}.env ./.env
 
+RUN yarn cache clean
```

Comment:
I'm not sure this is needed as this should be a clean node lts-alpine installation. But it shouldn't hurt anything